includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -619,11 +619,14 @@ public function register_scripts() {
 		if ( $this->supports( 'tokenization' ) ) {
 			$script_dependencies[] = 'woocommerce-tokenization-form';
 		}
-
+		$script_src_url               = plugins_url( 'dist/checkout.js', WCPAY_PLUGIN_FILE );
```

Comment:
Looks to me like a repeated piece of code in all the files. Can this be converted to a function or class?

Reply:
Pushed new code that replaced all these manual imports with function.